libs/providers/ofrep-web/src/lib/ofrep-web-provider.ts

```diff
@@ -260,37 +272,41 @@ export class OFREPWebProvider implements Provider {
   }
 
   /**
-   * Evaluate is a function retrieving the value from a flag in the cache.
+   * _resolve is a function retrieving the value from a flag in the cache.
    * @param flagKey - name of the flag to retrieve
    * @param type - type of the flag
+   * @param defaultValue - default value
    * @private
    */
-  private evaluate<T extends FlagValue>(flagKey: string, type: string): ResolutionDetails<T> {
+  private _resolve<T extends FlagValue>(flagKey: string, type: string, defaultValue: T): ResolutionDetails<T> {
     const resolved = this._flagCache[flagKey];
+
     if (!resolved) {
-      throw new FlagNotFoundError(`flag key ${flagKey} not found in cache`);
+      return {
+        value: defaultValue,
+        flagMetadata: this._flagSetMetadataCache,
+        reason: 'ERROR',
```

Comment:
nit: there's an error enum you could use.

Reply:
Yes